ouroboros-consensus-byron-test/src/Ouroboros/Consensus/ByronDual/Node/Serialisation.hs

```diff
@@ -99,11 +99,6 @@ instance DecodeDisk DualByronBlock (AnnTip DualByronBlock) where
 
 instance SerialiseNodeToNodeConstraints DualByronBlock
 
--- | @'HeaderHash' 'DualByronBlock'@
-instance SerialiseNodeToNode DualByronBlock ByronHash where
-  encodeNodeToNode _ _ = encodeByronHeaderHash
-  decodeNodeToNode _ _ = decodeByronHeaderHash
```

Comment:
Lets' get rid of `encodeByronHeaderHash` and `decodeByronHeaderHash` as well.

Reply:
These are still used internally.